OneCommunityGlobal/HGNRest
# Shuhua - add google doc icon

# Description
<img width="763" alt="description" src="https://github.com/OneCommunityGlobal/HGNRest/assets/83729360/36289f64-c46f-4bcf-8856-029a99c1924f">

## Related PRS (if any):

To test this backend PR you need to checkout the [PR1980](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/1980) frontend PR.

## Main changes explained:

- Added `adminLinks` field for each user in `getTasksForTeams()` in `src/helpers/taskHelper.js`

## How to test:

1. check into current branch
2. do `npm run build && npm start` to run this PR locally
3. checkout the frontend PR[#1980](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/1980)
